src/system-script.js

```diff
@@ -85,6 +84,12 @@ export function instantiate(load) {
 	return system
 		.import(address)
 		.then(load => {
+			if (system.loads && system.trace) {
+				// fix source tracing
+				setTimeout(() => {
```

Comment:
This needs to be in a timeout because the `loads` object isn't defined yet and gets overridden once the instantiate hook resolves.